drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -116,6 +118,10 @@ def enrollments(**args)
       resolve_enrollments(object.enrollments_including_wip, dangerous_skip_permission_check: true, **args)
     end
 
+    def assessments(**args)
+      resolve_assessments(object.custom_assessments, **args)
```

Comment:
Just noting that permissions are checked in the `HasAssessments` concern which applies the `viewable_by` scope. If the user doesn't have permission to view enrollments in the project, then this will turn up empty. (Unlike the above enrollment resolver which raises an exception).
I think that is fine but an improvement would probably to be to add a `dangerous_skip_permission_check` like we do above, so we can skip the intensive `viewable_by` scope.

Reply:
Yes that makes sense! I'll make the change on the `HasAssessments` concern